cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -97,12 +117,16 @@
         "genetic_ancestry_Indigenous_American",
         "genetic_ancestry_Oceanian",
         "genetic_ancestry_South_Asian",
+        "cell_line_ontology_term_id",
+        "perturbation_strategy",
+        "perturbation_id",
+        "perturbation_role",
+        "perturbation_target_gene_id",
     ],
 )
 
-good_obs["donor_id"] = good_obs["donor_id"].astype("category")
-good_obs["suspension_type"] = good_obs["suspension_type"].astype("category")
-good_obs["tissue_type"] = good_obs["tissue_type"].astype("category")
+for field in categorical_fields:
```

Comment:
since i was adding so many more categorical fields, it seemed easier to define the categorical fields above and map them to category columns in a for loop